src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -12,9 +12,26 @@ internal static class AnalyzerHelper
         private const string CSharpCompilerAnalyzerTypeName = "Microsoft.CodeAnalysis.Diagnostics.CSharp.CSharpCompilerDiagnosticAnalyzer";
         private const string VisualBasicCompilerAnalyzerTypeName = "Microsoft.CodeAnalysis.Diagnostics.VisualBasic.VisualBasicCompilerDiagnosticAnalyzer";
 
-        private const string AnalyzerExceptionDiagnosticId = "AD0001";
+        // These are the error codes of the compiler warnings. 
+        // Keep the ids the same so that de-duplication against compiler errors
+        // works in the error list (after a build).
+        internal const string WRN_AnalyzerCannotBeCreatedIdCS = "CS8032";
+        internal const string WRN_AnalyzerCannotBeCreatedIdVB = "BC42376";
+        internal const string WRN_NoAnalyzerInAssemblyIdCS = "CS8033";
+        internal const string WRN_NoAnalyzerInAssemblyIdVB = "BC42377";
+        internal const string WRN_UnableToLoadAnalyzerIdCS = "CS8034";
+        internal const string WRN_UnableToLoadAnalyzerIdVB = "BC42378";
+
+        // Shared with Compiler
+        internal const string AnalyzerExceptionDiagnosticId = "AD0001";
+        internal const string AnalyzerDriverExceptionDiagnosticId = "AD0002";
+
+        // IDE only errors
+        internal const string WRN_AnalyzerCannotBeCreatedId = "AD1000";
```

Comment:
Actually, I feel a better long term approach here is to make all the "WRN_AnalyzerXXX" kind of diagnostics language independent. They are reported in the core CodeAnalysis layer, but the only reason they have CS or VB prefix is they also need to run within command line compilers. We cannot then avoid all the language mapping code/IDs here.